zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/Helium.java

```diff
@@ -411,7 +412,7 @@ public HeliumPackageSuggestion suggestApp(Paragraph paragraph) {
    * @return ordered list of enabled buildBundle package
    */
   public List<HeliumPackage> getBundlePackagesToBundle() {
-    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfoWithoutRefresh();
+    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfo();
```

Comment:
Thanks @1ambda , I now understand this in more detail. Check my last commit. In the visualization order unit-test, I added an initial call to fetch package status. That allows the tests to pass. In the other 3 places, I was able to remove to non-refresh without breaking the tests so happy with this version. Can you review once again? I had merge conflicts against latest master so had to rebase... Thanks.